packages/dd-trace/src/priority_sampler.js

```diff
@@ -24,12 +24,12 @@ const DEFAULT_KEY = 'service:,env:'
 const defaultSampler = new Sampler(AUTO_KEEP)
 
 class PrioritySampler {
-  constructor (env, { sampleRate, rateLimit = 100, rules = [] } = {}) {
+  constructor (env, { sampleRate, rateLimit = 100, samplingRules = [] } = {}) {
     this._env = env
-    this._rules = this._normalizeRules(rules, sampleRate)
+    this._samplingRules = this._normalizeRules(samplingRules, sampleRate)
```

Comment:
This name change is redundant here since it's in a `sampler` class.

Reply:
100% - I had only done it for consistency in the new naming standardizing, even in the sampler, and since we will be adding span sampling rules down the line, but I can remove this change.